.changeset/sour-mangos-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+"@patternfly/pfe-badge": major
+---
+
+Breaking changes: Added BaseBadge and extended it as pfe-badge
```

Comment:
Rather than explaining the steps taken as a dev, explain what the major changes are to the user. 
~~~suggestion
Rewrote `<pfe-badge>` to more closely implement the PatternFly v4 spec. This includes several component API changes

#### Before
```html
Insert V1 example
```

#### After
```html
Insert V2 example
```
~~~

See [the docs](https://patternflyelements.org/components/badge) for more information.